packages/client/src/components/Router.tsx

```diff
@@ -1,28 +1,27 @@
 import React, { Suspense } from "react";
-import { BrowserRouter, Redirect, Route, Switch } from "react-router-dom";
+import { Redirect, Route, Switch } from "react-router-dom";
 
 import LoadingPage from "../pages/LoadingPage";
 import { redirectConfig, routeConfig } from "../routes";
 import ScrollToTop from "./ScrollToTop";
 
-const Router = () => {
-  return (
-    <BrowserRouter>
-      <ScrollToTop />
-      <Suspense fallback={<LoadingPage />}>
-        <Switch>
-          {redirectConfig.map((redirect) => (
-            <Redirect key={redirect.from} {...redirect} />
+const Router = () => (
+  // BrowserRouter has been moved to `../index.tsx` to make it accissible for other HOC's
+  <>
+    <ScrollToTop />
```

Comment:
Btw, do we still use the scroll to top?

Reply:
For sure. On every route. To reset the viewport.